TentativeVolunteers.md

```diff
@@ -1,4 +1,5 @@
 # Tentative Volunteers Setup
+![Tetative Volunteers Setup](https://images.theconversation.com/files/74422/original/image-20150311-20537-bym06g.jpg?ixlib=rb-1.1.0&q=45&auto=format&w=926&fit=clip)
```

Comment:
I cannot accept unrelated images in the volunteer's guide. We have a lot of content, we do not need unrelated distractions from that content.

I know your work is guided by this issue: https://github.com/the-laurel/laurels/issues/61#issue-964948276. In that issue it says:
> This guide should be enriched with images and animations
> The volunteer should use LiceCap to capture the animation of the screen for each of those steps and include them in that document as .gif images/animations
> The text of the guide should be detailed in such a way that each list item becomes a subchapter that describes in detail what is animated.

The animations and images should strictly be related to demonstrating the steps in the guide. E.g. animating how to install Metamask, how to get the public Ethereum address, how to make a PR, etc.